lib/api/BackbeatAPI.js

```diff
@@ -617,6 +620,38 @@ class BackbeatAPI {
         return cb(null, {});
     }
 
+    /**
+     * Validate that the POST request body has the necessary content
+     * @param {String} body - the POST request body string
+     * @return {Object} object containing an error and the request body
+     */
+    _parseScheduleResumeBody(body) {
+        const msg = 'The body of your POST request is not well-formed';
+        let reqBody;
+        const defaultRes = { hours: 6 };
+        if (!body) {
+            return defaultRes;
+        }
+        try {
+            reqBody = JSON.parse(body);
+            if (!reqBody.hours) {
+                return defaultRes;
+            }
+            if (reqBody.hours === '' || isNaN(reqBody.hours) ||
+                parseInt(reqBody.hours, 10) <= 0) {
```

Comment:
Having sub-hour granularity could be useful in some cases. (either allowing a floating-point value, or allowing a "minutes" param as well could do). Not for this PR.